src/lib/holocene/tooltip.svelte

```diff
@@ -104,7 +104,6 @@
             {copyIconTitle}
             clickAllToCopy
             content={text}
-            color="white"
```

Comment:
I removed `color` from `Copyable` since it uses an `IconButton` now (it looks like it wasn't working when it was using just an `Icon` either). 

Wondering how we might want to go about passing a different `stroke` color here or if it's even necessary?

Reply:
**Update**: Fixed in https://github.com/temporalio/ui/pull/1586/commits/e034d84f92064f6961cb181e0dc6c8a4a8a28632